Android.mk

```diff
@@ -0,0 +1,14 @@
+_has_qemu := $(filter qemu=%,$(BOARD_KERNEL_CMDLINE))
+ADDITIONAL_DEFAULT_PROPERTIES += \
```

Comment:
*ADDITIONAL_DEFAULT_PROPERTIES* marked as read-only and should not be used anywhere other than Android build system itself.